deployment/cdk/opensearch-service-migration/lib/migration-assistance-stack.ts

```diff
@@ -29,140 +27,119 @@ export class MigrationAssistanceStack extends Stack {
-        // migrationFargateTask.addVolume({
-        //     name: 'shared-traffic-comparator-volume',
+        // const crPolicyStatement = new PolicyStatement({
+        //     effect: Effect.ALLOW,
+        //     actions: ["ec2:DescribeSubnets",
+        //         "ec2:DescribeVpcs",
+        //         "ec2:DescribeSecurityGroups",
+        //         "ec2:DescribeRouteTables",
+        //         "ec2:DescribeVpcEndpoints",
+        //         "ec2:DescribeVpcAttribute",
+        //         "ec2:DescribeNetworkAcls",
+        //         "kafka:*"],
+        //     resources: ["*"]
+        // })
+        // const crPolicy = AwsCustomResourcePolicy.fromStatements([crPolicyStatement])
+        // const mskPublicEndpointCustomResource = new AwsCustomResource(this, 'MigrationMSKPublicEndpointCR', {
+        //     onCreate: {
+        //         service: 'Kafka',
+        //         action: 'updateConnectivity',
+        //         parameters: {
+        //             ClusterArn: mskCluster.attrArn,
+        //             CurrentVersion: "K3P5ROKL5A1OLE",
+        //             ConnectivityInfo: {
+        //                 PublicAccess: {
+        //                     Type: 'SERVICE_PROVIDED_EIPS'
+        //                 }
+        //             }
+        //         },
+        //         physicalResourceId: PhysicalResourceId.of(Date.now().toString())
+        //     },
+        //     policy: crPolicy,
+        //     vpc: props.vpc
+        // })
+
+        // const mskPublicEndpointCustomResource = new AwsCustomResource(this, 'MigrationMSKPublicEndpointCR', {
+        //     onCreate: {
+        //         service: 'Kafka',
+        //         action: 'getBootstrapBrokers',
+        //         parameters: {
+        //             ClusterArn: mskCluster.attrArn,
+        //             Outputs: {
+        //                 CustomOutput: 'customOutputValue'
+        //             }
+        //         },
+        //         physicalResourceId: PhysicalResourceId.of(Date.now().toString())
+        //     },
+        //     policy: AwsCustomResourcePolicy.fromSdkCalls({resources: [mskCluster.attrArn]}),
+        //     vpc: props.vpc
+        // })
+
+        // new CfnOutput(this, 'MSKBrokerOutput', {
+        //     value: mskPublicEndpointCustomResource.getResponseField("Outputs.CustomOutput")
         // });
 
-        // Create Fargate Service
-        const migrationFargateService = new FargateService(this, "migrationFargateService", {
-            cluster: ecsCluster,
-            taskDefinition: migrationFargateTask,
-            desiredCount: 1
+        const efsSecurityGroup = new SecurityGroup(this, 'efsSecurityGroup', {
+            vpc: props.vpc,
+            allowAllOutbound: true,
+        });
+        efsSecurityGroup.addIngressRule(efsSecurityGroup, Port.allTraffic());
+
+        // Create an EFS file system for the traffic-comparator
+        const fileSystem = new FileSystem(this, 'MigrationEFS', {
```

Comment:
We might need volumes for a number of different things - MigrationEFS isn't very descriptive.  Is this for sqlite?